README.md

```diff
@@ -1,154 +1,66 @@
-- Bencher uses GZIP and Benchmark uses ZSTD compression for writing parquet files
-- Test sourcees are in *src/it/java*
-
-### Bencher vs Benchmark for the same queries (rates are records/sec):
-Test Details
-- All tests run the same query that joins a table of 10 or 100 million records to a table of 100K records
-- Test Description: Rows are either released incrementally (auto increment) or read from a static parquet file (parquet static)
-- Bencher Rate: Rows processed per second while running the query in Bencher
-- Benchmark Rate: Rows processed per second while running the query in Benchmark
-
-|Test Description|Bencher Rate|Benchmark Rate|
-|----------------|------------|-----------|
-|stock join 10m auto increment|339430.53|363266.50|
-|stock join 10m parquet static|579667.06|569670.70|
-|stock join 100m auto increment|358337.90|398665.25|
-|stock join 100m parquet static|553502.34|693847.00|
-
-### Benchmark producing records to the Kafka broker and consumed by the Deephaven query script (rates are records/sec):
-Test Details
-- All tests produce data to Kafka while it is consumed/queried in Deephaven
-- First two tests run the same query that joins a table of 10 or 100 million records to a table of 100K records
-- Last test loads a table of 100 million rows from Kafka
-- Test Description: Test data is produced to Kafka and consumed in the query using Avro spec
-- Query Rate: Rows processed per second while running the query in Benchmark
-- Producer Rate: Rows produce to Kafka per seconds. 
-- If Query Rate is significantly lower than Producer Rate, Deephaven Engine isn't keeping up
-
-|Test Description|Query Rate|Producer Rate|
-|----------------|-----------|-------------|
-|stock join 10m kafka append|224411.48|415973.38|
-|stock join 100m kafka append|275232.62|384831.48|
-|consumer count 100m kafka append|441027.97|442944.34|
-
-
-
+# Deephaven Benchmark
+
+![Operation Rate Change Tracking By Release](docs/BenchmarkChangeTable.png)
+
+The Benckmark framework provides support for gathering performance measurements and statistics for operations on tabular data.  It uses the JUnit
+framework as a runner and works from popular IDEs or from the command line. It is geared towards scale testing interfaces capable of ingesting 
+table data, transforming it, and returning tabular results. 
+
+Currently, most benchmarks that use the framework are aimed at broad coverage of single query operations executed in 
+[Deephaven Core Community](https://deephaven.io/community/) through the Barrage Java Client. Tests focus on querying static parquet files, 
+streamed Kafka topics, and replayed data.
+
+The project maintains several hundred standardized benchmarks for Deephaven query operations that are tracked both from release-to-release and 
+nightly. Results are regularly published to a read-only GCloud bucket available through the public storage API 
+[deephaven-benchmark](https://storage.googleapis.com/deephaven-benchmark).  
```

Comment:
Yes.  The storage API only lists objects (directories, files) instead of a nice browse-able directory-like structure.  Maybe the link is unnecessary, since people would most likely use the provided query anyway?

Reply:
Removed the link, since it is dealt with later in PublishedResults.md